src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -108,7 +108,7 @@ struct
       | Error e ->
           Logger.warn logger ~module_:__MODULE__ ~location:__LOC__
             ~metadata:[("error", `String (Error.to_string_hum e))]
-            "Bad transaction snark: $error" ;
+            "Unable to verify proof with transaction snark: $error" ;
```

Comment:
Grammar is weird here. A proof doesn't *have* a transaction snark, it *is* a transaction snark. Well, an argument anyway. Also "Unable to" kind of implies it's our fault. How about: "Invalid transaction snark for statement $statement: $error"?